espnet/st/pytorch_backend/st.py

```diff
@@ -18,15 +18,19 @@
 from espnet.asr.asr_utils import adam_lr_decay
```

Comment:
The change seems to come from MD and ensemble.
Another possibility is to make another execution `espnet/st/pytorch_backend/st_md.py`  for MD (but not sure for the ensemble part).